### PR TITLE
Added  Main.java

### PR DESCRIPTION
**Pull request by osr-pafoca-test**
First file added

### CODE REVIEW

**Comment by osr-pafoca-test** [comment]
looks good

**Comment by osr-reviewcrawler-02** [comment]
looks good

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class Main{
+
+  public static final void main(String[] args){
+      // doSomething
```
Comment (osr-reviewcrawler-01) [inline Main.java:4]: not enough to review ...

Comment (osr-reviewcrawler-01) [inline Main.java:4]: another comment

**Comment by osr-reviewcrawler-01** [comment]
thats fine

**Pull request merged**